components/ui-api-layer/internal/gqlschema/schema.graphql

```diff
@@ -2,6 +2,8 @@
 
 scalar JSON
 
+scalar Labels
```

Comment:
@pkosiec please take a look and tell if it's ok to name it `Labels`
for me, it's quite ok and if future we need to rename other incorrect "labels" to `tag`